rules/windows/defense_evasion_suspicious_certutil_commands.toml

```
@@ -18,9 +43,55 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*", "endgame-*"
 language = "eql"
 license = "Elastic License v2"
 name = "Suspicious CertUtil Commands"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating Suspicious CertUtil Commands
+
+Certutil.exe is a command-line utility program that is included with Microsoft Windows operating systems. It is used to manage and manipulate digital certificates and certificate services on a Windows-based computer.
+
+Attackers can abuse `certutil.exe` utility to download and/or deobfuscate malware, offensive security tools, and certificates from external sources in order to take the next steps in a compromised environment, and this rule identifies command line arguments used to accomplish it.
```

Not sure if "behaviors" is the correct word here, or if it should be something else like "occurrences" or "events". 

```suggestion
Attackers can abuse `certutil.exe` utility to download and/or deobfuscate malware, offensive security tools, and certificates from external sources to take the next steps in a compromised environment. This rule identifies command line arguments used to accomplish these behaviors.
```